app/views/publishers/vacancies/_show.html.slim

```diff
@@ -5,11 +5,11 @@
     .share-url-info
       .govuk-grid-row
         .govuk-grid-column-one-half
-          .share-url.display-none class="govuk-!-margin-bottom-6"
-            h2.govuk-heading-m class="govuk-!-margin-top-0" | Share this job
+          .govuk-inset-text
```

Comment:
Actually I think this whole section should be deleted?

https://ukgovernmentdfe.slack.com/archives/CG9A8H1HP/p1626087365227700?thread_ts=1626086493.226200&cid=CG9A8H1HP

Reply:
absolutely right, i didnt follow up on this convo with the actual code. does say in the ticket requirements also